### PR TITLE
fix(FEC-7184): remove setDisableCustomPlaybackForIOS10Plus flag

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

This flag disables iOS's native fullscreen and causing problems if playinsline not set to true.

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/ima.js [hunk 1]
```diff
@@ -399,7 +399,6 @@ export default class Ima extends BasePlugin {
     this._sdk.settings.setPlayerVersion(VERSION);
     this._sdk.settings.setVpaidAllowed(true);
     this._sdk.settings.setVpaidMode(this._sdk.ImaSdkSettings.VpaidMode.ENABLED);
-    this._sdk.settings.setDisableCustomPlaybackForIOS10Plus(true);
```
Comment (OrenMe) [inline src/ima.js]: @dan-ziv if we have playsinline config then this is required, so let's make this somewhere in the form of:
`this._sdk.settings.setDisableCustomPlaybackForIOS10Plus(config.playsinline);`

need to make sure playback preformance with regards to the user gesture restriction in this case as well

Comment (dan-ziv) [reply src/ima.js]: @OrenMe 
From my checks this flag is not mandatory and IOS10Plus playback works just fine

Comment (dan-ziv) [reply src/ima.js]: this is just if we want to disable AV player full screen on iOS10Plus - do we want?

Comment (OrenMe) [reply src/ima.js]: this is what I meant in my comment - if player is set to work inline iOS10 then we need to set this as well, so need to align to player config, and also enable passing other options if we are on the subject, like the vpaidAllowed and vpaidMode.

Comment (dan-ziv) [reply src/ima.js]: no need to set this as well, playsinline on ios10plus works without this flag

**Pull request merged**